astro/src/content/docs/extend/code/lambdas/steam-reconcile.mdx

```diff
@@ -24,7 +24,7 @@ function reconcile(user, registration, userInfo) {
 This lambda must contain a function named `reconcile` that takes three parameters. The parameters that the lambda is passed are:
 
 <ReconcileLambdaUserRegistrationParameters />
-* `userInfo` - the JSON payload returned by the Steam Token Details API. This is read-only.
+* `userInfo` - the JSON payload returned by the Steam Token Details API. This object is read-only.
```

Comment:
This is actually the player summaries API. Per google we are the only folks that call it the Token Details API: https://www.google.com/search?q=steam+%22token+details+api%22&client=firefox-b-1-d&sca_esv=746687e9b17ac835&ei=9wYhZ5a8GpK_0PEPgJf26Q8&ved=0ahUKEwjWk7Sk-7OJAxWSHzQIHYCLPf0Q4dUDCA8&uact=5&oq=steam+%22token+details+api%22&gs_lp=Egxnd3Mtd2l6LXNlcnAiGXN0ZWFtICJ0b2tlbiBkZXRhaWxzIGFwaSIyBRAhGKABMgUQIRigATIFECEYoAEyBRAhGKABMgUQIRigATIFECEYqwJI9w9QkAZY1AxwAXgBkAEAmAGMAaABiQKqAQMwLjK4AQPIAQD4AQGYAgOgApgCwgIKEAAYsAMY1gQYR8ICCBAAGIAEGKIEmAMAiAYBkAYIkgcDMS4yoAfvBw&sclient=gws-wiz-serp

Reply:
Will update.